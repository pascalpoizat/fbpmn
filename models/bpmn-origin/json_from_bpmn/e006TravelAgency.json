{
    "isInterrupt": {},
    "catN": {
        "Task_1q91vog": "SendTask",
        "IntermediateThrowEvent_0neineb": "CatchMessageIntermediateEvent",
        "ExclusiveGateway_0i09ijx": "AndGateway",
        "IntermediateThrowEvent_12d113r": "CatchMessageIntermediateEvent",
        "ExclusiveGateway_192ovii": "XorGateway",
        "StartEvent_1": "NoneStartEvent",
        "StartEvent_1f3jj6d": "NoneStartEvent",
        "ExclusiveGateway_1dc5v3z": "XorGateway",
        "TravelAgency_": "Process",
        "EndEvent_0u6deep": "CatchMessageIntermediateEvent",
        "Task_07u875a": "ReceiveTask",
        "Task_1bn6n5q": "SendTask",
        "EndEvent_055yt9k": "NoneEndEvent",
        "EndEvent_10gqkzy": "TerminateEndEvent",
        "Customer_": "Process",
        "Task_002ndsu": "SendTask",
        "Task_1ne4gpy": "SendTask",
        "IntermediateThrowEvent_0xjpikb": "CatchMessageIntermediateEvent",
        "Task_1v9s881": "SendTask",
        "ExclusiveGateway_0wgdt1i": "XorGateway"
    },
    "catE": {
        "SequenceFlow_1h5h7h5": "NormalSequenceFlow",
        "SequenceFlow_0rfye55": "NormalSequenceFlow",
        "SequenceFlow_0sfyd5z": "NormalSequenceFlow",
        "SequenceFlow_1rlj8av": "NormalSequenceFlow",
        "SequenceFlow_13z4ilm": "NormalSequenceFlow",
        "SequenceFlow_0ljbxox": "NormalSequenceFlow",
        "SequenceFlow_0n80biv": "NormalSequenceFlow",
        "SequenceFlow_0mdvaai": "NormalSequenceFlow",
        "SequenceFlow_016h32p": "NormalSequenceFlow",
        "SequenceFlow_11rxzkm": "NormalSequenceFlow",
        "SequenceFlow_1uwq0b6": "ConditionalSequenceFlow",
        "MessageFlow_1m551dh": "MessageFlow",
        "MessageFlow_1yfhhru": "MessageFlow",
        "MessageFlow_04an7oz": "MessageFlow",
        "SequenceFlow_0b34324": "NormalSequenceFlow",
        "SequenceFlow_1qku5do": "NormalSequenceFlow",
        "SequenceFlow_1rma3l8": "NormalSequenceFlow",
        "SequenceFlow_0b6ku63": "DefaultSequenceFlow",
        "MessageFlow_1goz1mt": "MessageFlow",
        "MessageFlow_0knd10s": "MessageFlow",
        "SequenceFlow_1dptcxp": "NormalSequenceFlow",
        "SequenceFlow_1fn4lqy": "NormalSequenceFlow",
        "SequenceFlow_00s838q": "NormalSequenceFlow"
    },
    "containN": {
        "TravelAgency_": [
            "StartEvent_1f3jj6d",
            "ExclusiveGateway_1dc5v3z",
            "Task_1bn6n5q",
            "ExclusiveGateway_0i09ijx",
            "IntermediateThrowEvent_0xjpikb",
            "EndEvent_10gqkzy",
            "Task_1ne4gpy",
            "Task_002ndsu",
            "IntermediateThrowEvent_0neineb"
        ],
        "Customer_": [
            "Task_1v9s881",
            "ExclusiveGateway_192ovii",
            "ExclusiveGateway_0wgdt1i",
            "StartEvent_1",
            "Task_07u875a",
            "EndEvent_055yt9k",
            "EndEvent_0u6deep",
            "IntermediateThrowEvent_12d113r",
            "Task_1q91vog"
        ]
    },
    "messageE": {
        "MessageFlow_1m551dh": "Confirmation",
        "MessageFlow_1yfhhru": "Travel",
        "MessageFlow_04an7oz": "Ticket",
        "MessageFlow_1goz1mt": "Payment",
        "MessageFlow_0knd10s": "Offer"
    },
    "timeInformation": {},
    "targetE": {
        "SequenceFlow_1h5h7h5": "IntermediateThrowEvent_12d113r",
        "SequenceFlow_0rfye55": "IntermediateThrowEvent_0xjpikb",
        "SequenceFlow_0sfyd5z": "ExclusiveGateway_192ovii",
        "SequenceFlow_1rlj8av": "IntermediateThrowEvent_0neineb",
        "SequenceFlow_13z4ilm": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_0ljbxox": "EndEvent_0u6deep",
        "SequenceFlow_0n80biv": "Task_002ndsu",
        "SequenceFlow_0mdvaai": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_016h32p": "Task_07u875a",
        "SequenceFlow_11rxzkm": "EndEvent_10gqkzy",
        "SequenceFlow_1uwq0b6": "Task_1v9s881",
        "MessageFlow_1m551dh": "EndEvent_0u6deep",
        "MessageFlow_1yfhhru": "IntermediateThrowEvent_0xjpikb",
        "MessageFlow_04an7oz": "IntermediateThrowEvent_12d113r",
        "SequenceFlow_0b34324": "Task_1bn6n5q",
        "SequenceFlow_1qku5do": "EndEvent_055yt9k",
        "SequenceFlow_1rma3l8": "ExclusiveGateway_0wgdt1i",
        "SequenceFlow_0b6ku63": "ExclusiveGateway_0wgdt1i",
        "MessageFlow_1goz1mt": "IntermediateThrowEvent_0neineb",
        "MessageFlow_0knd10s": "Task_07u875a",
        "SequenceFlow_1dptcxp": "Task_1q91vog",
        "SequenceFlow_1fn4lqy": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_00s838q": "Task_1ne4gpy"
    },
    "name": "e006TravelAgency",
    "attached": {},
    "containE": {
        "TravelAgency_": [
            "SequenceFlow_11rxzkm",
            "SequenceFlow_00s838q",
            "SequenceFlow_0n80biv",
            "SequenceFlow_1rlj8av",
            "SequenceFlow_0rfye55",
            "SequenceFlow_13z4ilm",
            "SequenceFlow_0mdvaai",
            "SequenceFlow_0b34324",
            "SequenceFlow_1fn4lqy"
        ],
        "Customer_": [
            "SequenceFlow_1uwq0b6",
            "SequenceFlow_0b6ku63",
            "SequenceFlow_0sfyd5z",
            "SequenceFlow_016h32p",
            "SequenceFlow_1rma3l8",
            "SequenceFlow_1dptcxp",
            "SequenceFlow_1h5h7h5",
            "SequenceFlow_0ljbxox",
            "SequenceFlow_1qku5do"
        ]
    },
    "edges": [
        "MessageFlow_0knd10s",
        "MessageFlow_1yfhhru",
        "MessageFlow_1m551dh",
        "MessageFlow_1goz1mt",
        "MessageFlow_04an7oz",
        "SequenceFlow_1uwq0b6",
        "SequenceFlow_0b6ku63",
        "SequenceFlow_0sfyd5z",
        "SequenceFlow_016h32p",
        "SequenceFlow_1rma3l8",
        "SequenceFlow_1dptcxp",
        "SequenceFlow_1h5h7h5",
        "SequenceFlow_0ljbxox",
        "SequenceFlow_1qku5do",
        "SequenceFlow_11rxzkm",
        "SequenceFlow_00s838q",
        "SequenceFlow_0n80biv",
        "SequenceFlow_1rlj8av",
        "SequenceFlow_0rfye55",
        "SequenceFlow_13z4ilm",
        "SequenceFlow_0mdvaai",
        "SequenceFlow_0b34324",
        "SequenceFlow_1fn4lqy"
    ],
    "messages": [
        "Offer",
        "Travel",
        "Confirmation",
        "Payment",
        "Ticket"
    ],
    "sourceE": {
        "SequenceFlow_1h5h7h5": "Task_1q91vog",
        "SequenceFlow_0rfye55": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_0sfyd5z": "Task_07u875a",
        "SequenceFlow_1rlj8av": "IntermediateThrowEvent_0xjpikb",
        "SequenceFlow_13z4ilm": "ExclusiveGateway_0i09ijx",
        "SequenceFlow_0ljbxox": "IntermediateThrowEvent_12d113r",
        "SequenceFlow_0n80biv": "IntermediateThrowEvent_0neineb",
        "SequenceFlow_0mdvaai": "Task_1bn6n5q",
        "SequenceFlow_016h32p": "ExclusiveGateway_0wgdt1i",
        "SequenceFlow_11rxzkm": "Task_1ne4gpy",
        "SequenceFlow_1uwq0b6": "ExclusiveGateway_192ovii",
        "MessageFlow_1m551dh": "Task_002ndsu",
        "MessageFlow_1yfhhru": "Task_1v9s881",
        "MessageFlow_04an7oz": "Task_1ne4gpy",
        "SequenceFlow_0b34324": "ExclusiveGateway_1dc5v3z",
        "SequenceFlow_1qku5do": "EndEvent_0u6deep",
        "SequenceFlow_1rma3l8": "StartEvent_1",
        "SequenceFlow_0b6ku63": "ExclusiveGateway_192ovii",
        "MessageFlow_1goz1mt": "Task_1q91vog",
        "MessageFlow_0knd10s": "Task_1bn6n5q",
        "SequenceFlow_1dptcxp": "Task_1v9s881",
        "SequenceFlow_1fn4lqy": "StartEvent_1f3jj6d",
        "SequenceFlow_00s838q": "Task_002ndsu"
    },
    "nodes": [
        "Customer_",
        "TravelAgency_",
        "Task_1v9s881",
        "ExclusiveGateway_192ovii",
        "ExclusiveGateway_0wgdt1i",
        "StartEvent_1",
        "Task_07u875a",
        "EndEvent_055yt9k",
        "EndEvent_0u6deep",
        "IntermediateThrowEvent_12d113r",
        "Task_1q91vog",
        "StartEvent_1f3jj6d",
        "ExclusiveGateway_1dc5v3z",
        "Task_1bn6n5q",
        "ExclusiveGateway_0i09ijx",
        "IntermediateThrowEvent_0xjpikb",
        "EndEvent_10gqkzy",
        "Task_1ne4gpy",
        "Task_002ndsu",
        "IntermediateThrowEvent_0neineb"
    ],
    "nameN": {
        "Task_1q91vog": "Pay Travel",
        "IntermediateThrowEvent_0neineb": "Payment Received",
        "IntermediateThrowEvent_12d113r": "Ticket Received",
        "ExclusiveGateway_192ovii": "is the offer interesting?",
        "StartEvent_1": "Offer Management",
        "StartEvent_1f3jj6d": "Offer Needed",
        "EndEvent_0u6deep": "Booking Confirmed",
        "Task_07u875a": "Check Travel Offer",
        "Task_1bn6n5q": "Make Travel Offer",
        "EndEvent_055yt9k": "Transaction Completed",
        "EndEvent_10gqkzy": "Offer Completed",
        "Task_002ndsu": "Confirm Booking",
        "Task_1ne4gpy": "Order Ticket",
        "IntermediateThrowEvent_0xjpikb": "Booking Received",
        "Task_1v9s881": "Book Travel"
    }
}