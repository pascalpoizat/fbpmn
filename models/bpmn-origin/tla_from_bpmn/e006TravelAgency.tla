---------------- MODULE e006TravelAgency ----------------

EXTENDS TLC, PWSTypes

VARIABLES nodemarks, edgemarks, net

Interest ==
  "Customer_" :> { "Ticket", "Offer", "Confirmation" }
  @@ "TravelAgency_" :> { "Payment", "Travel" }

ContainRel ==
  "Customer_" :> { "Task_1v9s881", "ExclusiveGateway_192ovii", "ExclusiveGateway_0wgdt1i", "StartEvent_1", "Task_07u875a", "EndEvent_055yt9k", "EndEvent_0u6deep", "IntermediateThrowEvent_12d113r", "Task_1q91vog" }
  @@ "TravelAgency_" :> { "StartEvent_1f3jj6d", "ExclusiveGateway_1dc5v3z", "Task_1bn6n5q", "ExclusiveGateway_0i09ijx", "IntermediateThrowEvent_0xjpikb", "EndEvent_10gqkzy", "Task_1ne4gpy", "Task_002ndsu", "IntermediateThrowEvent_0neineb" }

Node == { "Customer_", "TravelAgency_", "Task_1v9s881", "ExclusiveGateway_192ovii", "ExclusiveGateway_0wgdt1i", "StartEvent_1", "Task_07u875a", "EndEvent_055yt9k", "EndEvent_0u6deep", "IntermediateThrowEvent_12d113r", "Task_1q91vog", "StartEvent_1f3jj6d", "ExclusiveGateway_1dc5v3z", "Task_1bn6n5q", "ExclusiveGateway_0i09ijx", "IntermediateThrowEvent_0xjpikb", "EndEvent_10gqkzy", "Task_1ne4gpy", "Task_002ndsu", "IntermediateThrowEvent_0neineb" }

Edge == { "MessageFlow_0knd10s", "MessageFlow_1yfhhru", "MessageFlow_1m551dh", "MessageFlow_1goz1mt", "MessageFlow_04an7oz", "SequenceFlow_1uwq0b6", "SequenceFlow_0b6ku63", "SequenceFlow_0sfyd5z", "SequenceFlow_016h32p", "SequenceFlow_1rma3l8", "SequenceFlow_1dptcxp", "SequenceFlow_1h5h7h5", "SequenceFlow_0ljbxox", "SequenceFlow_1qku5do", "SequenceFlow_11rxzkm", "SequenceFlow_00s838q", "SequenceFlow_0n80biv", "SequenceFlow_1rlj8av", "SequenceFlow_0rfye55", "SequenceFlow_13z4ilm", "SequenceFlow_0mdvaai", "SequenceFlow_0b34324", "SequenceFlow_1fn4lqy" }

Message == { "Offer", "Travel", "Confirmation", "Payment", "Ticket" }

msgtype ==
   "MessageFlow_0knd10s" :> "Offer"
@@ "MessageFlow_1yfhhru" :> "Travel"
@@ "MessageFlow_1m551dh" :> "Confirmation"
@@ "MessageFlow_1goz1mt" :> "Payment"
@@ "MessageFlow_04an7oz" :> "Ticket"


source ==
   "MessageFlow_0knd10s" :> "Task_1bn6n5q"
@@ "MessageFlow_1yfhhru" :> "Task_1v9s881"
@@ "MessageFlow_1m551dh" :> "Task_002ndsu"
@@ "MessageFlow_1goz1mt" :> "Task_1q91vog"
@@ "MessageFlow_04an7oz" :> "Task_1ne4gpy"
@@ "SequenceFlow_1uwq0b6" :> "ExclusiveGateway_192ovii"
@@ "SequenceFlow_0b6ku63" :> "ExclusiveGateway_192ovii"
@@ "SequenceFlow_0sfyd5z" :> "Task_07u875a"
@@ "SequenceFlow_016h32p" :> "ExclusiveGateway_0wgdt1i"
@@ "SequenceFlow_1rma3l8" :> "StartEvent_1"
@@ "SequenceFlow_1dptcxp" :> "Task_1v9s881"
@@ "SequenceFlow_1h5h7h5" :> "Task_1q91vog"
@@ "SequenceFlow_0ljbxox" :> "IntermediateThrowEvent_12d113r"
@@ "SequenceFlow_1qku5do" :> "EndEvent_0u6deep"
@@ "SequenceFlow_11rxzkm" :> "Task_1ne4gpy"
@@ "SequenceFlow_00s838q" :> "Task_002ndsu"
@@ "SequenceFlow_0n80biv" :> "IntermediateThrowEvent_0neineb"
@@ "SequenceFlow_1rlj8av" :> "IntermediateThrowEvent_0xjpikb"
@@ "SequenceFlow_0rfye55" :> "ExclusiveGateway_0i09ijx"
@@ "SequenceFlow_13z4ilm" :> "ExclusiveGateway_0i09ijx"
@@ "SequenceFlow_0mdvaai" :> "Task_1bn6n5q"
@@ "SequenceFlow_0b34324" :> "ExclusiveGateway_1dc5v3z"
@@ "SequenceFlow_1fn4lqy" :> "StartEvent_1f3jj6d"

target ==
   "MessageFlow_0knd10s" :> "Task_07u875a"
@@ "MessageFlow_1yfhhru" :> "IntermediateThrowEvent_0xjpikb"
@@ "MessageFlow_1m551dh" :> "EndEvent_0u6deep"
@@ "MessageFlow_1goz1mt" :> "IntermediateThrowEvent_0neineb"
@@ "MessageFlow_04an7oz" :> "IntermediateThrowEvent_12d113r"
@@ "SequenceFlow_1uwq0b6" :> "Task_1v9s881"
@@ "SequenceFlow_0b6ku63" :> "ExclusiveGateway_0wgdt1i"
@@ "SequenceFlow_0sfyd5z" :> "ExclusiveGateway_192ovii"
@@ "SequenceFlow_016h32p" :> "Task_07u875a"
@@ "SequenceFlow_1rma3l8" :> "ExclusiveGateway_0wgdt1i"
@@ "SequenceFlow_1dptcxp" :> "Task_1q91vog"
@@ "SequenceFlow_1h5h7h5" :> "IntermediateThrowEvent_12d113r"
@@ "SequenceFlow_0ljbxox" :> "EndEvent_0u6deep"
@@ "SequenceFlow_1qku5do" :> "EndEvent_055yt9k"
@@ "SequenceFlow_11rxzkm" :> "EndEvent_10gqkzy"
@@ "SequenceFlow_00s838q" :> "Task_1ne4gpy"
@@ "SequenceFlow_0n80biv" :> "Task_002ndsu"
@@ "SequenceFlow_1rlj8av" :> "IntermediateThrowEvent_0neineb"
@@ "SequenceFlow_0rfye55" :> "IntermediateThrowEvent_0xjpikb"
@@ "SequenceFlow_13z4ilm" :> "ExclusiveGateway_1dc5v3z"
@@ "SequenceFlow_0mdvaai" :> "ExclusiveGateway_0i09ijx"
@@ "SequenceFlow_0b34324" :> "Task_1bn6n5q"
@@ "SequenceFlow_1fn4lqy" :> "ExclusiveGateway_1dc5v3z"

CatN ==
   "Customer_" :> Process
@@ "TravelAgency_" :> Process
@@ "Task_1v9s881" :> SendTask
@@ "ExclusiveGateway_192ovii" :> ExclusiveOr
@@ "ExclusiveGateway_0wgdt1i" :> ExclusiveOr
@@ "StartEvent_1" :> NoneStartEvent
@@ "Task_07u875a" :> ReceiveTask
@@ "EndEvent_055yt9k" :> NoneEndEvent
@@ "EndEvent_0u6deep" :> CatchMessageIntermediateEvent
@@ "IntermediateThrowEvent_12d113r" :> CatchMessageIntermediateEvent
@@ "Task_1q91vog" :> SendTask
@@ "StartEvent_1f3jj6d" :> NoneStartEvent
@@ "ExclusiveGateway_1dc5v3z" :> ExclusiveOr
@@ "Task_1bn6n5q" :> SendTask
@@ "ExclusiveGateway_0i09ijx" :> Parallel
@@ "IntermediateThrowEvent_0xjpikb" :> CatchMessageIntermediateEvent
@@ "EndEvent_10gqkzy" :> TerminateEndEvent
@@ "Task_1ne4gpy" :> SendTask
@@ "Task_002ndsu" :> SendTask
@@ "IntermediateThrowEvent_0neineb" :> CatchMessageIntermediateEvent

CatE ==
   "MessageFlow_0knd10s" :> MessageFlow
@@ "MessageFlow_1yfhhru" :> MessageFlow
@@ "MessageFlow_1m551dh" :> MessageFlow
@@ "MessageFlow_1goz1mt" :> MessageFlow
@@ "MessageFlow_04an7oz" :> MessageFlow
@@ "SequenceFlow_1uwq0b6" :> ConditionalSeqFlow
@@ "SequenceFlow_0b6ku63" :> DefaultSeqFlow
@@ "SequenceFlow_0sfyd5z" :> NormalSeqFlow
@@ "SequenceFlow_016h32p" :> NormalSeqFlow
@@ "SequenceFlow_1rma3l8" :> NormalSeqFlow
@@ "SequenceFlow_1dptcxp" :> NormalSeqFlow
@@ "SequenceFlow_1h5h7h5" :> NormalSeqFlow
@@ "SequenceFlow_0ljbxox" :> NormalSeqFlow
@@ "SequenceFlow_1qku5do" :> NormalSeqFlow
@@ "SequenceFlow_11rxzkm" :> NormalSeqFlow
@@ "SequenceFlow_00s838q" :> NormalSeqFlow
@@ "SequenceFlow_0n80biv" :> NormalSeqFlow
@@ "SequenceFlow_1rlj8av" :> NormalSeqFlow
@@ "SequenceFlow_0rfye55" :> NormalSeqFlow
@@ "SequenceFlow_13z4ilm" :> NormalSeqFlow
@@ "SequenceFlow_0mdvaai" :> NormalSeqFlow
@@ "SequenceFlow_0b34324" :> NormalSeqFlow
@@ "SequenceFlow_1fn4lqy" :> NormalSeqFlow

LOCAL preEdges ==
  [ i \in {} |-> {}]
PreEdges(n,e) == preEdges[n,e]

PreNodes(n,e) == { target[ee] : ee \in preEdges[n,e] }
          \union { nn \in { source[ee] : ee \in preEdges[n,e] } : CatN[nn] \in { NoneStartEvent, MessageStartEvent } }

BoundaryEvent ==
  [ i \in {} |-> {}]

WF == INSTANCE PWSWellFormed
ASSUME WF!WellTyped
ASSUME WF!WellFormedness

ConstraintNode == TRUE \* none
ConstraintEdge == TRUE \* none
Constraint == TRUE     \* none
INSTANCE PWSConstraints
INSTANCE UserProperties
INSTANCE PWSSemantics

================================================================

