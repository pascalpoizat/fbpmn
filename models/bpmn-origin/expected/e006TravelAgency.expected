Working in /tmp/e006TravelAgency.7oCIA with 1 worker(s)
[92mtransformation done[0m
<<"Processes=", 2, "Nodes=", 20, "Gateway=", 4, "SF=", 18, "MF=", 5>>
---------- Network01Bag ----------
[X] Prop01Type
     states=470 trans=966 depth=43
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=470 trans=966 depth=
[X] Prop04MsgSound
     states=470 trans=966 depth=43
---------- Network02FifoPair ----------
[X] Prop01Type
     states=474 trans=881 depth=40
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=474 trans=881 depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network03Causal ----------
[X] Prop01Type
     states=806 trans=1398 depth=41
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states= trans= depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network04Inbox ----------
[X] Prop01Type
     states=474 trans=881 depth=40
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states= trans= depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network05Outbox ----------
[X] Prop01Type
     states=474 trans=881 depth=40
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states= trans= depth=
[ ] Prop04MsgSound
     states=301 trans=558 depth=
---------- Network06Fifo ----------
[X] Prop01Type
     states=522 trans=932 depth=40
[ ] Prop02Safe
     states= trans= depth=10
[ ] Prop03Sound
     states=522 trans=932 depth=
[ ] Prop04MsgSound
     states= trans= depth=
---------- Network07RSC ----------
[X] Prop01Type
     states=247 trans=420 depth=38
[ ] Prop02Safe
     states= trans= depth=14
[ ] Prop03Sound
     states=247 trans=420 depth=
[ ] Prop04MsgSound
     states=247 trans=420 depth=
done.
